src/main/java/picard/sam/AbstractAlignmentMerger.java

```diff
@@ -86,6 +87,7 @@
     public static final int MAX_RECORDS_IN_RAM = 500000;
 
     private static final char[] RESERVED_ATTRIBUTE_STARTS = {'X', 'Y', 'Z'};
+    private static final int MAX_CROSS_SPECIES_CONTAM_WARN = 1_000;
```

Comment:
Since Java 8.  Syntax isn't supported in scala though (but there you can write `1e3.toInt`).